examples/siro_sandbox/README_VR.md

```diff
@@ -0,0 +1,218 @@
+# VR Human-in-the-loop (HITL) Evaluation
+
+**Note: VR HITL evaluation currently only works from this branch.**
+
+HITL evaluation can be driven from VR. In this mode, the sandbox app acts as a server that can be remotely accessed by a client.
+
+As it stands, the VR integration can only be used with the `fetch` app state. In this mode, the user controls a human avatar that can play fetch with a policy-driven Spot robot.
+
+## Table of Contents
+- [VR Human-in-the-loop (HITL) Evaluation](#vr-human-in-the-loop-hitl-evaluation)
+  - [Table of Contents](#table-of-contents)
+  - [Installation](#installation)
+    - [Requirements](#requirements)
+      - [Server](#server)
+      - [Client](#client)
+  - [Launch Commands](#launch-commands)
+      - [Example](#example)
+  - [Unity Data Folder](#unity-data-folder)
+    - [Dataset Processing](#dataset-processing)
+      - [Installation](#installation-1)
+      - [Usage](#usage)
+      - [Importing Data Into Unity](#importing-data-into-unity)
+  - [Connection](#connection)
+  - [Server Controls](#server-controls)
+  - [Troubleshooting](#troubleshooting)
+    - [Deployment to the VR Headset](#deployment-to-the-vr-headset)
+    - [Connection Issues](#connection-issues)
+    - [Slow Performance](#slow-performance)
+  - [Other information](#other-information)
+    - [Serving HTTPS content from your local machine](#serving-https-content-from-your-local-machine)
+
+
+## Installation
+
+The system is composed of the following components:
+
+* The **Server**, which is the sandbox app.
+* The **Client**, which runs on Unity and is deployed on a VR headset.
+
+### Requirements
+
+#### Server
+
+| Requirements | Notes |
+|---|---|
+| `habitat-sim` | Use the `main` branch. Bullet is required. |
+| `habitat-lab` | This specific `habitat-lab` version must be installed. See [instructions](../../README.md#installation). The `main` branch is currently incompatible. |
+| Datasets | `python -m habitat_sim.utils.datasets_download --uids habitat_humanoids hab_spot_arm ycb hssd-hab --data-path data/` |
+| [hssd-models](https://huggingface.co/datasets/hssd/hssd-models) | Required for dataset processing. |
```

Comment:
Aside from pointing folks at these repos, what exactly do they need to do? clone them to a specific location locally?

Reply:
At some point I want to get away from putting application data at `habitat-lab/data` . Instead, it should go at the root directory of the application.